Dockerfile

```diff
@@ -19,11 +20,19 @@ RUN apk update && apk add \
     && tar -xzf "/tmp/v${BATS_VERSION}.tar.gz" -C /tmp/ \
     && bash "/tmp/bats-${BATS_VERSION}/install.sh" /usr/local \
     \
+    && wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && tar -C /usr/local/bin -xzvf dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    && rm dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
+    \
     && rm -rf /var/cache/apk/* && rm -rf /tmp/*
 
+
 ## Configure Service
 
-COPY install/main.dist.cf /etc/postfix/main.cf
+ENV CERTIFICATE_PUBLIC=/etc/postfix/cert/smtp.cert
+ENV CERTIFICATE_PRIVATE=/etc/postfix/cert/smtp.key
```

Comment:
Can we support inline data here?

For example:
```shell
read -r -d '' CERTIFICATE_PUBLIC <<'EOF'
... CERT
... HERE
EOF
```

This would be able to let me set it in docker cloud web manager directly, without care about any files.

Reply:
It would be a bit hacky probably. How would you differentiate between a path and inline data? Because it starts with a `/`? I would rather use another variable and in the `init-openssl.sh` echo a `CERTIFICATE_PUBLIC_PAYLOAD` to the `CERTIFICATE_PUBLIC` path. Same with private.

It's rare that both CERTIFICATE_PUBLIC and CERTIFICATE_PUBLIC_PAYLOAD be defined at the same time. But I think different variables would end up in cleaner expectation.

Reply:
On more thought, is it realistic to manually edit the certificate content? In my use case either in docker cloud or in rancher a letsencrypt container periodically renew the certificate. Been able to mount the output as is in the smf container is enough to keep things working.

If a renewal of the certificate is the motivation for this, it will force you to manually be involved.

And on other side, isn't it to risky to allow save the private key in the database of the docker cloud manager you use?

I think that easy manual editing of the content of the certificate can be obtained by this PR as is plus using for example https://filebrowser.github.io/ which can be used as a docker container and has inline file editing.